arcade_machine/games/main_menu.py

```diff
@@ -1,37 +1,115 @@
-from pygame import KEYDOWN, K_w, K_s, K_RETURN
+import pygame.sprite
```

Comment:
This class had turned into a bloated mess. Not going to try reading this. We'll have to just discuss this one.

Reply:
That is insulting. It is actually quite clean. I will add comments for you.